packages/core/src/middleware/defaults/feeEstimator.ts

```diff
@@ -18,7 +18,7 @@ export const defaultFeeEstimator: <C extends MiddlewareClient>(
     // for more information about maxFeePerGas and maxPriorityFeePerGas
 
     const feeData = await client.estimateFeesPerGas();
-    if (!feeData.maxFeePerGas || !feeData.maxPriorityFeePerGas) {
+    if (!feeData.maxFeePerGas || feeData.maxPriorityFeePerGas == null) {
```

Comment:
think I left a comment about this somewhere else, but nit:

```suggestion
    if (feeData.maxFeePerGas == null || feeData.maxPriorityFeePerGas == null) {
```

Reply:
we should be consistent

Reply:
hmm this was because maxFeePerGas cannot be 0, but maxPriorityFeePerGas can be

Reply:
yea that's fine, if one can be 0 and not the other checking both explicitly for `== null` still covers the one that can't be 0 and at least they read the same.

it's nit, so it's fine either way